colibre/registration.py

```diff
@@ -175,6 +215,24 @@
         ),
     )
 
+# Get HI to dust ratio
+try:
+    dust_to_hi_ratio = total_dust_mass / HI_mass
+    dust_to_hi_ratio.name = "M_{{\\rm dust}}/M_{{\\rm HI}}"    
+    setattr(self, "gas_dust_to_hi_ratio", dust_to_hi_ratio)
+    
+except AttributeError:
+    # We did not produce these quantities.
+    setattr(
+        self,
+        "gas_dust_to_hi_ratio",
+        unyt.unyt_array(
+            catalogue.masses.m_gas**0,
```

Comment:
Same comments here as above, would be nice to have the units explicit.

Reply:
it is the correct units, but put an explicit conversion in each one